lib/lib_webgpu_dawn.cpp

```diff
@@ -622,8 +623,19 @@ WGPU_DEVICE_LOST_REASON Dawn_to_WGPU_DEVICE_LOST_REASON[] = {
 
 //////////////////////////////////////////////////////////////////////
 
+#ifdef DAWN_UNSAFE_API
+const char* enabledToggles = "allow_unsafe_apis";
+
+WGPUDawnTogglesDescriptor instanceTogglesDesc{ { nullptr, WGPUSType_DawnTogglesDescriptor}, 1, &enabledToggles };
+WGPUInstanceDescriptor instanceDesc{ reinterpret_cast<WGPUChainedStruct*>(&instanceTogglesDesc) };
```

Comment:
Can you add `static` to the above three global objects so they retain .cpp local linkage?